torch/_functorch/config.py

```diff
@@ -59,7 +62,10 @@
 # or default config to true and fix relevant bugs
 from torch._inductor.config import is_fbcode
 
-view_replay_for_aliased_outputs = not is_fbcode()
+# View replay is currently not compatible with AOTAutogradCache, since
+# FunctionalTensors are not serializable. We'll need to make them
+# serializable before enabling warm cache with this config turned on.
+view_replay_for_aliased_outputs = (not is_fbcode()) and (not enable_autograd_cache)
```

Comment:
fyi @ysiraichi @JackCaoG 

We're working on caching above the AOTAutograd layer as part of the effort to lower compile times, and right now this doesn't play well with Yukio's view-replay logic.

The problem is that we need to serialize any metadata that we need at runtime. Right now, the view-replay logic relies on grabbing the `std::vector<ViewMeta>` in `FunctionalTensorWrapper` at runtime in order to know what views to replay, but this whole thing is not serializable.

We're probably going to have to re-work the view-replay logic to save something else to the `ViewAndMutationMeta` that is more serializable (not the entire `FunctionalTensor`).